SCAutolib/cli_commands.py

```diff
@@ -3,37 +3,146 @@
 """
 
 import click
+from pathlib import Path
+from sys import exit
+
+from SCAutolib import logger, exceptions, schema_user, ReturnCode
 from SCAutolib.controller import Controller
 
 
 @click.group()
-def cli():
-    pass
+@click.option("--conf", "-c",
+              required=True,
```

Comment:
Is `required` needed when there is a default value?

Reply:
Not realy